### PR TITLE
[4.0] clean up jui folder

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
Move files around (rename and remove the jquery. part from the name)

### CODE REVIEW

File: administrator/components/com_languages/views/override/tmpl/edit.php [hunk 1]
```diff
@@ -17,11 +17,11 @@
 
 $expired = ($this->state->get('cache_expired') == 1 ) ? '1' : '';
 
-JHtml::_('stylesheet', 'overrider/overrider.css', array('version' => 'auto', 'relative' => true));
+JHtml::_('stylesheet', 'com_overrider/overrider.css', array('version' => 'auto', 'relative' => true));
```
Comment (brianteeman) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: shouldnt the foldername be com_languages? there is no component called com_overrider

File: administrator/components/com_languages/views/override/tmpl/edit.php [hunk 2]
```diff
@@ -17,11 +17,11 @@
...
 
 JHtml::_('behavior.core');
 JHtml::_('jquery.framework');
-JHtml::_('script', 'overrider/overrider.min.js', array('version' => 'auto', 'relative' => true));
+JHtml::_('script', 'com_overrider/overrider.min.js', array('version' => 'auto', 'relative' => true));
```
Comment (brianteeman) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: as above

File: administrator/components/com_languages/views/override/tmpl/edit.php (hunk 1 above)
Comment (wilsonge) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: :+1:

Comment (dgt41) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: done

File: administrator/components/com_languages/views/override/tmpl/edit.php (hunk 2 above)
Comment (dgt41) [inline administrator/components/com_languages/views/override/tmpl/edit.php]: done

**Pull request merged**